wgsl/index.bs

```diff
@@ -3745,24 +3745,23 @@ A WGSL texture corresponds to a WebGPU {{GPUTexture}}.
 
 A texture is either arrayed, or non-arrayed:
 
-* A <dfn noexport>non-arrayed texture</dfn> is a grid of texels. Each texel has a unique grid coordinate.
+* A <dfn noexport>non-arrayed texture</dfn> is a grid of texels.
 * An <dfn noexport>arrayed texture</dfn> is a homogeneous array of grids of texels.
-    In an arrayed texture, each texel is identified with its unique combination of array index and grid coordinate.
 
 A texture has the following features:
 
 : texel format
 :: The data in each texel. See [[#texel-formats]].
-:  dimensionality
+:  <dfn dfn-for=texture noexport>dimensionality</dfn>
```

Comment:
I'm not a big fan of the mixed definitions here. Some are definitions, some are not. Is there a way to make this all more consistent? For example, define texel format here and link to it in the formats section.

Reply:
Yeah. there's more cleanup that's needed.  e.g. we don't really say what a sampled texture is, and warn that a *multisampled* texture is not sampled. (Gotcha!)
And that stands out in this PR now that it's been broken out of its parent PR.